.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: $DOCKER_USERNAME/katex-fonts:29d4cdbc
+
+    steps:
+      - checkout
+
+      - run:
+          name: Build fonts
+          command: |
+            cd src
+            cp default.cfg custom.cfg
+            make custom.cfg.pl
+            make -C fonts/OTF/TeX ttf woff woff2
+            cd ..
+            rm -f fonts/*.*
+            cp src/fonts/OTF/TeX/ttf/*.ttf fonts
+            cp src/fonts/OTF/TeX/woff/*.woff fonts
+            cp src/fonts/OTF/TeX/woff2/*.woff2 fonts
+            tar zcf fonts.tar.gz fonts
+
+      - run:
+          name: Build metrics
+          command: ./buildMetrics.sh
+
+      - store_artifacts:
+          path: fonts.tar.gz
+          destination: fonts.tar.gz
```

Comment:
Why bother storing artifacts if we're using this repo as a submodule?

Reply:
I guess if we moved font creation into the main repo, we could avoid checking the built fonts in and instead grab the most recent artifact so that people don't have to build fonts themselves when doing KaTeX development.

Reply:
@kevinbarabash It's like screenshots. CircleCI doesn't commit files and they have to be manually committed.